llumnix/entrypoints/vllm/api_server.py

```diff
@@ -171,6 +176,28 @@ async def stream_results() -> AsyncGenerator[bytes, None]:
     ret = {"text": text_outputs}
     return JSONResponse(ret)
 
+def init_per_token_latency_breakdown_dict() -> Dict[str, int]:
```

Comment:
consider using a namedtuple

Reply:
Since the latency should be returned to the client in json response format, and the usage of dict is more consistent with json, so i think dict might be better.